### DISCUSSION TITLE
Recreational Data

### DISCUSSION THREAD

**Discussion by melissamonk-NOAA** [pfmc-assessments/california-data#2] [General]
Migrated from the GDrive:

**Q (MHM-SWFSC):** How to separate out catches landed in the U.S. but caught in Mexican waters for PR and PC modes? 
**A (AS-CDFW):** CDFW does not send catch estimates from Mexican waters to RecFIN, therefore the CTE001 report (for the most part) contains no catch estimates from Mexican waters. Note that CRFS does not sample CPFV trips that fished Mexican waters. CTE001 contains some catch estimates from Mexican waters for years 2005 - 2013 when PSMFC produced the CRFS estimates. These estimates can be identified by RecFIN Water_Area Name = Mexico in the CTE001 report.

**Q  (CRW-NWFSC)** While CDFW may not report catch estimates that occurred in Mexican waters, it is my understanding that there are biological samples that can be from Mexican waters. In the past I have used the AGENCY_WATER_AREA_NAME to filter out these records, is this correct? Should other filters be also applied to remove records that should not be used (e..g., samples from AGENCY_WATER_AREA_NAME = NOT KNOWN or/and INLAND)?
**A (AS-CDFW):** Correct, AGENCY_WATER_AREA_NAME can be used to filter by water area, including Mexico, in the RecFIN SD001 Biological Detail report.

**Q (MHM-SWFSC):** What data are available from the southern CA [Vickie Wine](https://aquadocs.org/bitstream/handle/1834/18270/Admin._report_No._82-7.pdf?sequence=1&isAllowed=y) PR and dive survey from the 1970s?    Are these considered confidential? 
**A (JEP-CDFW):** I have been trying to find out if this data is entered anywhere. It is really old and was done using computer punch cards. It was pre-excel and all the analysis and reports were already completed. I have found links to a number of the reports but I don’t know if the data contained is useful for stock assessments. I made an excel document summarizing the available reports and where to find them. The document has been uploaded into the Rec folder. We spoke with Vicki Wine and she believes that the original punch cards and data were disposed of sometime in the late 80’s or early 90’s. The project had ended, MRFSS essentially took its place, and no one was using the data from the Vickie Wine Survey. I have asked some other CDFW staff if they have copies of the raw data, but I am not hopeful I will find anything other than the summarized reports. I believe all that remains of that project is the info found in the quarterly/annual reports.
https://docs.google.com/spreadsheets/d/1JT4_VvKeFJahkAOd3_NYGFxcLo6lYooX/edit#gid=863832667 

**Q (JEP-CDFW)** these reports were done in conjunction with a federal grant. I am guessing the reports are the only products NMFS would have. Is there a possibility NMFS would have a copy of the raw data if that was a requirement of the funding? 

**Q (BJL-NWFSC):** How do we determine discard   quantities within MRFSS data? Ive heard MRFSS data has B4 data but in the past have not found that field. Thus, Ive calculated mortality rate using the proportion of AB1 catches that are B1. Using the fields I had at the time, the formula was (TSP_HARV) / (TSP_HARV + TSPCLAIM). Is there a better way to do this?

**A: (JB- CDFW):** In some cases, they are treated as all dead.  There is no proportion of catch by depth to apply discard mortality rates by in most years since depth data was not collected until 1999.  We do have data from 1999 and 2000 to base proportions on prior to closures in 2001.  We can use the proportion of catch by depth for the RecFISH model north and south of Conception based on those proportions and multiply by the percent mortality in each bin for surface release, then sum for an average mortality rate for discards.  Problem is the B1 contains otherwise unavailable dead fish ie buddy drove off with the cooler, eaten by seagull, etc. that are not subject to discard mortality.  Historical onboard data could be used to ferret that out a bit, but it might be polishing peanuts. Alternatively, they may be very low irrespective and be considered negligible, but maybe not.  The other rate that was used up until about 2008 was 42% discard mortality applied to all rockfish species under early CRFS based on some back of the envelope calculations by PSMFC.

**Comment by brianlangseth-NOAA** [comment]
@JohnBudrick, @AshokCDFW
Two rec data questions:
1. Is there any issue with missed or unavailable data if I pull rec data for both CRFS and MRFSS data from the RecFIN website? I've pulled the data and want to ensure I'm not missing important components. The alternative would be for the state to pull the data and provide.
2. Is there any special treatment of the data depending on the value in SOURCE_CODE? Can both CPFV and BIO be used for compositions? I'm aware that CPFV applies to the charter mode, and not private.

**Reply by AshokCDFW** [reply]
All existing CRFS catch estimates should be available from RecFIN (with observed kept and unobserved kept catch lumped together). CDFW can additionally provide Excel files of annual estimates for each mode and year that CDFW produced the estimates (2015 through 2022) with observed kept catch and unobserved kept catch separated. The file “California Estimates and Data Source 2003-2022 (11-18-2022).xlsx” shows which months/modes are missing estimates. These months/modes will be missing regardless of whether the data are pulled from RecFIN or obtained from CDFW. CDFW’s Groundfish Project can provide proxy estimates for the April through June 2020 period where catch estimates are missing due to COVID sampling restrictions.

**Reply by JohnBudrick** [reply]
The CPFV entries are for individuals observed while sampling onboard and correspond to discarded fish that are measured prior to release on each drift for a subset of anglers, while BIO data is for observed fish whose lengths were recorded at the end of a fishing trip on the way back to the dock onboard for the PC mode or dockside for PC or PR.  Canary rockfish retention has been prohibited since 2001 and the onboard CPFV samples have been collected since 2003 providing a random sampling of discarded fish of all sizes in the PC mode.  Given that retention was prohibited until 2016, and there is limited data available otherwise to represent the composition of the catch apart from fish retained in the PR mode illegally, the combined composition of retained and discarded fish from the CPFV entries may provide a representative sample, though sampled at a different rate than the BIO fish that are retained and recorded.  Further consideration of the utility of the CPFV fish to represent the fishery or use of estimated discards vs. retained fish by numbers could be used to weight contributions in forming a composite composition including both retained and discarded fish in the PC mode.  The PR composition before prohibition can be compared to the PC to examine whether they were fishing on fish of the same size etc. to decide whether the PC compositions might be a representative proxy for PR.

**Comment by brianlangseth-NOAA** [comment]
@andreCDFW You mentioned about new ageing samples yesterday. The notes say they are from 2022. Are these aged already or are there plans to age them?

**Reply by andreCDFW** [reply]
@brianlangseth-NOAA None of our CDFW collected otoliths have been aged. For previous cycles and this cycle, we send our otoliths up to Patrick in Newport to be read. I mentioned our 2022 collections from the rec fishery yesterday (n=179) and wanted to mention we also have samples collected from our 2019 Commercial Pilot Project where we collected 147 and 47 from hook-and-line and trawl landings, respectively. I am happy to discuss these collections in further detail with you and the STAT, though it would likely be best to do so during a side meeting.

**Comment by EJDick-NOAA** [comment]
Background: CRFS catch estimates for 2004 are not currently available on the RecFIN website, and we are planning a call to discuss use of the 2004 CRFS catch estimates that were stored in the MRFSS legacy tables. However, CRFS *length* data for 2004 (and even earlier) are on the RecFIN website.
Question: @AshokCDFW @JohnBudrick Can one of you or someone else at CDFW describe the difference between the two sources of 2004 length data (MRFSS legacy table and the CRFS data currently online)? Which source should we use for this stock assessment cycle?

**Reply by JohnBudrick** [reply]
From the downloads I did for canary rockfish, I can see length data for as far back as 2003 in the CRFS era download (SD501) for discarded fish measured onboard CPFVs and for 2004 both retained and discarded fish lengths are present.  The lengths for retained observed fish from the MRFSS download are only provided up to 2003.

**Comment by brianlangseth-NOAA** [comment]
@JohnBudrick and others: Is there a MRFSS metadata table where I can understand the values within a field (e.g. understand what the values 1, 2, 6, and 7 are for MODE_FX)?

**Reply by JohnBudrick** [reply]
1 is man made
2 is beach and bank
6 is Party Charter
7 is Private Rental

Ill try to dig up the documentation of codes, but they are not included in the corresponding metadata for MRFSS

**Reply by EJDick-NOAA** [reply]
@brianlangseth-NOAA and @JohnBudrick Here is a pdf I downloaded in December of 2008 with metadata from the old RecFIN website.
[RecFIN_Websites_Instructions_Metadata_20081215.pdf](https://github.com/pfmc-assessments/california-data/files/10459866/RecFIN_Websites_Instructions_Metadata_20081215.pdf)

**Comment by brianlangseth-NOAA** [comment]
Conversation with @JohnBudrick today about MRFSS lengths. Sampling prior to 1993 used total length (T_LEN) followed by a period of both total and fork to generate conversions, and ending with fork length (LNGTH). Can mostly determine whether lengths were measured or converted using a regression based on decimals

**Comment by brianlangseth-NOAA** [comment]
@JohnBudrick In the opening post above, I don't understand the answer to the last question about MRFSS discard data. Does the MRFSS catch dataset include any discard information, either as a estimate of the magnitude or as an estimate of the rate? If so, what field(s) do I use to calculate discards?

**Reply by JohnBudrick** [reply]
It includes retained fish (A fish), as well as discarded dead and discarded live and otherwise unavailable dead fish with various dispositions that are all rolled up in the b1 fish.  Thus we use the sum of A and B1 to obtain total mortality for 1980-2003 when the discards in B1 were not broken out.  We called them all dead at that point, which likely overestimated mortality, but the relative contribution has prevented  anyone to trying to parse them out.  You can look at the proportion A vs B1 to get an idea of potential effects of assuming the B1 are all dead vs. the current day practice of applying discard mortality rates accounting for depth of release etc since 2009.  From around 2005 or so to 2008 they applied a 42% mortality rate to discards if memory serves.  Point is, we dont have the data in the historical period 1980-2004 to parse the discards and apply discard mortality rates.  We usually take the MRFSS estimates for AB1 to be representative of mortality either way.

**Reply by brianlangseth-NOAA** [reply]
Thanks @JohnBudrick. I will move forward with WGT_AB1_mt as total mortality.

**Comment by brianlangseth-NOAA** [comment]
@JohnBudrick While finalizing catch decisions for quillback for MRFSS years, I notice that estimates for PC mode in 1993-1995 are low (i.e. they are zero). This issue came up for Canary rockfish too. Based on the quillback STAT's discussion, PC sampling didn't commence until 1996 north of Point Conception (though perhaps it was north of San Luis Obispo county). Can you confirm? 

The implications are that in addition to the data gap in 1990-1992, there also is a gap for PC in 1993-1995, and that decisions should be made about filling in both. 

If indeed there is a gap, my plan is to use a similar approach as was used for Canary rockfish, namely to calculate an average ratio of PC:PR from other MRFSS years, and approximate PC mortality in 1993-1995 by multiplying the ratio by PR mortality for those years.

**Comment by JohnBudrick** [comment]
Hi Brian,

I understand there to have been a delay in resuming the PC sampling and thus estimates.  The ratio of PR to PC would be one way, but effort estimation for either back then is questionable potentially affecting results since it was the result of a phone coastal phone survey back then. The other option would be some type of interpolation with PC alone.  Perhaps see how the two compare and use the one you believe to be best informed.  If the ratio of PC to PR is fairly stable for the five years before and after, that might be the best option.  Interpolations with so many years uninformed between may be less optimal, but if there is a great deal of variability in the ratio of PC to PR due to effort estimation variability, maybe it could be more reasonable to just use the PC interpolation.  Just my two cents.  More than one way to address this.  Either way, the impact on the outcome doesn't seem likely to be enough to linger on it too much.  Use what is most justifiable given the considerations at hand.

Thanks,

John
